mitxpro/templates/base.html

```diff
@@ -1,5 +1,5 @@
 {% load static hijack_tags render_bundle %}
-{% load wagtailcore_tags render_notification %}
+{% load wagtailcore_tags render_notification string_utils %}
```

Comment:
instead of `string_utils`, we can put tags directly along with name convention under `../templatetags/`. 
`string_utils.py` ~change to ~ `startswith.py`

Reply:
I wanted to leave the library named as `string_utils` because we might end up adding more tags/filters later on. Similar to the `render_notification` library which actually provides the `latest_notification` tag.

Reply:
I prefer, to be specific in terms of name, we have a directory `templatetags` that should contains the relevant tags for better understand-ability

Reply:
Do you recommend I rename `render_notification` library to `latest_notification` as well?

Reply:
makes sense to me.